src/bin/nydus-image/core/feature.rs

```diff
@@ -45,6 +45,7 @@ impl TryFrom<&str> for Feature {
     fn try_from(f: &str) -> std::result::Result<Self, Self::Error> {
         match f {
             "blob_toc" => Ok(Self::BlobToc),
```

Comment:
Snapshotter converter has changed to `blob-toc`, we can remove it safely.

Reply:
@jiangliu It seems the `blob_toc` still be checked in `src/bin/nydus-image/main.rs`.

Reply:
Fixed by changing to "blob-toc".